mava/configs/logger/logger.yaml

```diff
@@ -1,23 +1,41 @@
 # --- Logging options ---
+base_exp_path: results/
 
-base_exp_path: results # Base path for logging.
-use_console: True # Whether to log to stdout.
-use_tb: False # Whether to use tensorboard logging.
-use_json: False # Whether to log marl-eval style to json files.
-use_neptune: False  # Whether to log to neptune.ai.
+loggers:
+  console:
+    _target_: mava.utils.logger.ConsoleLogger
+    enabled: True
+  neptune:
+    _target_: mava.utils.logger.NeptuneLogger
+    enabled: True
 
-# --- Other logger kwargs ---
-kwargs:
-  neptune_project: Instadeep/mava-benchmark
-  neptune_tag: [delete]
-  neptune_group_tag: [delete]
-  detailed_neptune_logging: False  # having mean/std/min/max can clutter neptune so we make it optional
-  json_path: ~ # If set, json files will be logged to a set path so that multiple experiments can
-    # write to the same json file for easy downstream aggregation and plotting with marl-eval.
-  upload_json_data: False # Whether JSON file data should be uploaded to Neptune for downstream
+    # If specified will resume the run with this run ID.
+    # This is useful for resuming runs and logging from multiple processes.
+    # NOTE: it will overwrite the run unless you set the timestep correctly.
+    run_id: ~
+    project: Instadeep/mava-benchmark
+    tag: [delete]
+    group_tag: [delete]
+    detailed_logging: False  # having mean/std/min/max can clutter neptune so we make it optional
+    architecture_name: ${arch.architecture_name}
+    # Whether JSON file data should be uploaded to Neptune for downstream
```

Comment:
I really like that the arguments for each logger are now directly associated with it. 
Just a note, can you please add a comment saying that we need the arch name for the neptune logger?

Reply:
Quite a nit point as well, but I find it easier if multi-line comments start on the same line as the config setting they are describing.

Reply:
Done for the arch, I gotta disagree with you on the comments though :joy:. Not too fussed though if you're passionate about it I'll change it